setup.py

```diff
@@ -360,13 +360,8 @@ def get_extensions():
         has_ffmpeg = False
     if has_ffmpeg:
         try:
-            # this splits on both dots and spaces as the output format differs across versions / platforms
-            ffmpeg_version_str = str(subprocess.check_output(["ffmpeg", "-version"]))
-            ffmpeg_version = re.split(r"ffmpeg version |\.| |-", ffmpeg_version_str)[1:3]
-            ffmpeg_version = ".".join(ffmpeg_version)
-            if StrictVersion(ffmpeg_version) >= StrictVersion('4.3'):
-                print(f'ffmpeg {ffmpeg_version} not supported yet, please use ffmpeg 4.2.')
-                has_ffmpeg = False
+            # This is to check if ffmpeg is installed properly.
+            _ = subprocess.check_output(["ffmpeg", "-version"])
```

Comment:
Nit suggestion below. Also, should we simplify the `except` statement to only catch a `CalledProcessError` below?

```suggestion
            subprocess.check_output(["ffmpeg", "-version"])
```

Reply:
Makes sense.